src/uu/snice/src/snice.rs

```diff
@@ -0,0 +1,299 @@
+// This file is part of the uutils procps package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+use std::{collections::HashSet, path::PathBuf, str::FromStr};
+
+use action::{perform_action, process_snapshot, users, ActionResult, SelectedTarget};
+use clap::{arg, crate_version, value_parser, Arg, ArgMatches, Command};
+use prettytable::{format::consts::FORMAT_CLEAN, row, Table};
+use priority::Priority;
+use sysinfo::Pid;
+use uu_pgrep::process::{ProcessInformation, Teletype};
+#[cfg(target_family = "unix")]
+use uucore::signals::ALL_SIGNALS;
+use uucore::{
+    error::{UResult, USimpleError},
+    format_usage, help_about, help_usage,
+};
+
+const ABOUT: &str = help_about!("snice.md");
+const USAGE: &str = help_usage!("snice.md");
+
+mod action;
+mod priority;
+
+#[derive(Debug)]
+enum SignalDisplay {
+    List,
+    Table,
+}
+
+#[allow(unused)]
+impl SignalDisplay {
+    fn try_new(matches: &ArgMatches) -> Option<SignalDisplay> {
```

Comment:
My suggestion is to use `new` as function name because the `try` prefix is usually used for functions that return a `Result`.